debian/control

```diff
@@ -44,8 +46,15 @@ Depends: ${misc:Depends},
     trawldb,
     wl-clipboard,
     xwayland
-Provides: regolith-desktop-session
```

Comment:
Why did you remove this?  (This is a virtual package specifier.  It is referenced [here](https://github.com/regolith-linux/regolith-desktop/blob/debian/debian/control#L15) as a way to say "install the flashback session or another package that provides the same virtual package)

Reply:
Actually, providing the `regoliith-desktop-session` virtual package causes `regolith-session-flashback` and `regolith-session-sway` to conflict. So, both packages cannot be installed together without removing the line.

Reply:
Huh, not sure how that would be true as I don't find a `Conflicts` cause but will merge for now and see if I can repro it